docs/library/stochastic/randomchoice/index.md

```diff
@@ -6,7 +6,7 @@ description: RandomChoice: Pick a random value from the given array. If a clock
 # RandomChoice
 
 ```python
-RandomChoice(values=std::vector<float> ( ), clock=None, reset=None)
+RandomChoice(values=std : : vector <float >(), clock=None, reset=None)
```

Comment:
Some auto-formatting is causing these colons to be broken up, which shouldn't happen. I remember seeing this before but can't remember the cause. Can you merge your branch with latest `master` and see if it's still happening?

Reply:
Having trouble with merging my branch to latest `master` -- might be worth going through this together on Friday.